src/ci/docker/x86_64-gnu-llvm-3.7/Dockerfile

```diff
@@ -26,5 +26,6 @@ ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 
 ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
-      --llvm-root=/usr/lib/llvm-3.7
+      --llvm-root=/usr/lib/llvm-3.7 \
+      --enable-test-miri
```

Comment:
Could we keep Travis pretty lean and run this only on the aux builder? I'm not sure why we'd want to run *this specific test suite* out of all our test suites on Travis?